chat-plugins/info.js

```diff
@@ -264,50 +264,55 @@ exports.commands = {
 	ipsearchall: 'ipsearch',
 	hostsearch: 'ipsearch',
 	ipsearch: function (target, room, user, connection, cmd) {
-		if (!target.trim()) return this.parse('/help ipsearch');
+		if (!target.trim()) return this.parse(`/help ipsearch`);
 		if (!this.can('rangeban')) return;
-		let results = [];
 
+		target = this.splitTargetText(target).trim();
+		let tarRoom = target.length ? Rooms(target.trim()) : null;
```

Comment:
...just call it `targetRoom`.

Reply:
Or even something clearer, like `inRoom`.

Reply:
Or `filterRoom`.

Reply:
I went with `targetRoom`